### PR TITLE
Make multiple PendingIntents distinguishable

### PR DESCRIPTION
**Pull request by Unrud**
If notifications for multiple new devices or folders are shown, the last Intent was used for all of them.

### CODE REVIEW

File: src/main/java/com/nutomic/syncthingandroid/syncthing/EventProcessor.java [hunk 1]
```diff
@@ -116,7 +117,8 @@ public void onEvent(long id, String type, JSONObject data) throws JSONException
                         .putExtra(SettingsActivity.EXTRA_IS_CREATE, true)
                         .putExtra(FolderFragment.EXTRA_DEVICE_ID, deviceId)
                         .putExtra(FolderFragment.EXTRA_FOLDER_ID, folderId);
-                pi = PendingIntent.getActivity(mContext, 0, intent, 0);
+                pi = PendingIntent.getActivity(mContext, (deviceId + folderId).hashCode(),
+                        intent, 0);
```
Comment (Nutomic) [inline src/main/java/com/nutomic/syncthingandroid/syncthing/EventProcessor.java]: Thanks, I didn't even know that was necessary. However, using a hashcode is definitely a hack. So can you add a comment similar to the one in line 199?

**Comment by Unrud** [comment]
> However, using a hashcode is definitely a hack. So can you add a comment similar to the one in line 199?

Done.

**Comment by Nutomic** [comment]
Can you rebase on master please?

**Comment by Unrud** [comment]
> Can you rebase on master please?

Done.

**Comment by Nutomic** [comment]
Thanks, merged :)

**Pull request merged**